about/updates/roadmap/index.md

```diff
@@ -22,29 +22,31 @@ We know it’s helpful for our partners at federal agencies to know what’s com
 This roadmap shows what we are working on now, and what we plan to be working after that. We’ll update this roadmap quarterly.
 
 
-## FY22, Q1: October to December 2021
+## FY22, Q2: January to March 2022
 
 ### Now
 
+* **Faceted Search: User Discovery**
+  * We will be conducting focus groups and initial research to feed into our development plans for faceted search.
 * **Version Upgrades**
   * We will be upgrading our Rails, ElasticSearch, and Ruby versions to keep them up-to-date and compliant. 
-* **Search.gov website redesign**
-  * We will be launching a new design that aligns with the [US Web Design System](https://designsystem.digital.gov/), in accordance with the [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/). The new site includes improved navigation and expanded documentation.
 * **Continuity of Operations Plan Improvements** 
   * We are taking steps to bolster our system's resiliance and reliability.
-* **Privacy Improvements**
-  * We are scoping ways to improve the privacy of user-inputted data into our search boxes
 * **New Developer Onboarding**
   * We are onboarding additional developers in support of our FY22 upcoming projects
+* **Legacy Code Removal**
+  * We are removing deprecated code in order to pave our path for new feature development this year.
+* **Search.gov website redesign - performance monitoring**
```

Comment:
I'm on the fence for including this, leaning towards removing it, since it's just admin team-side.